commands/sync.go

```diff
@@ -72,15 +73,16 @@ func writeDestinations(destinations map[string]string) {
 }
 
 func writeMonitors(monitors map[string]monitor.Monitor) {
-	destinationsPath := filepath.Join(rootDir, "monitors.yaml")
-	file, err := os.OpenFile(destinationsPath, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
-	check(err)
-	defer file.Close()
-	var monitorsList []monitor.Monitor
+	monitorsPath := filepath.Join(rootDir, "monitors/")
+	if _, err := os.Stat(monitorsPath); os.IsNotExist(err) {
+		os.Mkdir(monitorsPath, 0755)
+	}
 	for name := range monitors {
-		monitorsList = append(monitorsList, monitors[name])
+		monitorFile := filepath.Join(monitorsPath, sanitize.BaseName(name) + ".yaml")
+		file, err := os.OpenFile(monitorFile, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+		check(err)
+		data, err := yaml.Marshal(monitors[name])
+		check(err)
+		file.Write(data)
```

Comment:
So we'll be creating new file for each new monitor ?

Reply:
Yep that's right. We have about 30-40 monitors per env and they are pretty lengthy, so it made sense to keep them separated out.

Reply:
So initial my thoughts on sync is for just to start with all existing monotirs form cluster and organize however users  would want.

Reply:
Meaning let users choose whether they want one big yaml file vs a yaml file per monitor?